src/persistence/entities/applicationData.entity.js

```diff
@@ -0,0 +1,21 @@
+'use strict'
+
+const BaseModel = require('./base.entity')
+
+class ApplicationData extends BaseModel {
+  static get dynamicsEntity () {
+    return 'defra_webdata'
+  }
+
+  static get mapping () {
+    return [
+      { field: 'id', dynamics: 'defra_webdataid' },
+      { field: 'applicationId', dynamics: '_defra_applicationid_value', bind: { id: 'defra_applicationId', relationship: 'defra_defra_application_defra_webdata', dynamicsEntity: 'defra_applications' } },
+      { field: 'data', dynamics: 'defra_data', encode: true }
```

Comment:
This field could have a max length of 20,000 defined to prevent accidental attempts at insertion of arbitrary strings.

Reply:
Added max length